modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -377,6 +380,7 @@ public String toParamName(String name) {
 
     @Override
     public String toModelName(String name) {
+        name = super.toModelName(name);
```

Comment:
in this super method (DefaultCodegen.toModelName) I added the actual logic of adding suffix and prefix. Of course I could instead duplicate this here instead of calling the super method (like what Jim did in the C# version).